schema/shared/src/test/scala/zio/blocks/schema/ReflectSpec.scala

```diff
@@ -619,4 +635,18 @@ object ReflectSpec extends ZIOSpecDefault {
   implicit val rightSchema: Schema[Right[Int, Long]] = Schema.derived
   val eitherReflect: Reflect.Variant[Binding, Either[Int, Long]] =
     Schema.derived[Either[Int, Long]].reflect.asVariant.get
+  sealed trait PaymentMethod
+  case object Cash extends PaymentMethod {
+    lazy implicit val schema: Schema[Cash.type] = Schema.derived
```

Comment:
I had to make these lazy as I was getting runtime init exceptions - I tried to make them strict various different combinations but did not succeed